commit acd80bfd51fb16e53993ed3d9f6b658dba519272
Author: Antonio Russo <antonio.e.russo@gmail.com>
Date:   Mon Aug 17 22:29:51 2020 -0600

    Create SYSCONFDEFDIR configure parameter
    
    Provide downstream packagers with a tunable describing the location of
    the file containing environment variables to pass to the startup script.
    
    Defaults to the existing value, /etc/sysconfig.
    
    Signed-off-by: Antonio Russo <antonio.e.russo@gmail.com>

Forwarded: https://github.com/mchehab/rasdaemon/pull/22
Index: rasdaemon-0.6.7/configure.ac
===================================================================
--- rasdaemon-0.6.7.orig/configure.ac
+++ rasdaemon-0.6.7/configure.ac
@@ -175,6 +175,12 @@ AC_ARG_WITH(sysconfdefdir,
     ["/etc/sysconfig"])
 AC_SUBST([SYSCONFDEFDIR])
 
+AC_ARG_WITH(sysconfdefdir,
+    AC_HELP_STRING([--with-sysconfdefdir=DIR], [rasdaemon environment file dir]),
+    [SYSCONFDEFDIR=$withval],
+    [/etc/sysconfig])
+AC_SUBST([SYSCONFDEFDIR])
+
 AC_DEFINE([RAS_DB_FNAME], ["ras-mc_event.db"], [ras events database])
 AC_SUBST([RAS_DB_FNAME], ["ras-mc_event.db"])
 
